README.md

```diff
@@ -1,57 +1,50 @@
-    ended-at: 1594948069000
+    event: CI
+    artifact: Docker://my-org/my-repo/artifactId
+    run-status: ${{ job.status }}                     # possible values - Success, Failed, Canceled
+    run-started-at: 1594938057000
+    run-ended-at: 1594948069000
 ```
 
-> :clipboard: Note: Whilst the `build-pipeline-id` is optional, it is recommended to provide one to uniquely identify this workflow in Faros since multiple GitHub workflows can have the same name. If the `build-pipeline-id` is not provided it will be generated from the workflow name in the lowercase format `GITHUB_ORG/REPO/GITHUB_WORKFLOW_NAME`, e.g `my-org/my-repo/deployment`.
+### Report a code deployment (CD Event) To Faros
 
-### Report a Deployment Event To Faros CI/CD Deployment Model
-
-To report a build event to Faros specify `deploy` in the `model` parameter and the deployment details. To ensure the build is correctly linked to the build, provide the build model keys, i.e. `build-id`, `build-pipeline-id`, `build-org-id`, `build-source`.
+To report an artifact deployment to Faros specify `CD` in the `event` parameter and include the `CD` required fields.
 
 ```yaml
-- name: Report deployment info to Faros
-  id: emit-deployment-info
-  uses: faros-ai/faros-cicd-github-action@main
+- name: Report deployment to Faros
+  id: send-cd-event
+  uses: faros-ai/faros-cicd-github-action@v2.0.0
   with:
     api-key: ${{ secrets.FAROS_API_KEY }}
-    api-url: ${{ env.FAROS_API_URL }}
-    model: deploy
-    deploy-id: deploymentId
-    deploy-app-name: MyService               # name of the application being deployed
-    deploy-app-platform: ECS                 # platform application is deployed on
-    deploy-platform: CodeDeploy              # system used to orchestrate the deployment
-    build-id: build-id
-    build-pipeline-id: build-deploy-workflow
-    build-org-id: my-org
-    build-source: GitHub
-    started-at: 1594938057000
-    status: Queued                           # possible values - Canceled, Failed, Queued, Running, Success
+    event: CD
+    artifact: Docker://my-org/my-repo/artifactId
+    deploy: CodeDeploy://MyService/<env>/deploymentId # possible env values - Dev, Prod, Staging, QA
+    deploy-app-platform: ECS                          # the platform your application is deployed on
+    deploy-status: Success                            # possible values - Success, Failed, Canceled
+    deploy-started-at: 1594938057000
+    deploy-ended-at: 1594938059000
+    run-status: ${{ job.status }}                     # possible values - Success, Failed, Canceled
+    run-started-at: 1594938057000
+    run-ended-at: 1594948069000
 ```
 
-> :clipboard: Note: If you have both the report build and report deployment steps in the same workflow you can use the outputs from the report build step as the inputs for the deployment step build parameters, e.g. `${{ steps.emit-build-info.outputs.build-id }}`. See [action.yml](action.yml) for all available outputs.
```

Comment:
Why remove this?

Reply:
This action no longer has any outputs. Do we want this action to still output things? If so what would you like to output? We would need to parse that from the output of the `faros_event.sh` script which might be a little tricky.

Reply:
would it output the error at least?

Reply:
All of the CLI output is written to the console as you would expect (sent objects, response objects, errors etc), but no, currently there are no GitHub action step output values that could be used by later steps.